src/site/content/en/progressive-web-apps/promote-install/index.md

```diff
@@ -0,0 +1,382 @@
+description: |
+  How to promote installation of Progressive Web Apps and best practices.
+tags:
+  - post
+draft: true
+---
+
+Progressive Web Apps (PWAs) are a pattern for creating app-like, instant
+loading, reliable and installable websites and why would you want a user to
+install your web app? The same reason you'd want a user to install your native
+app from any app store.
+
+Users who install are your most engaged audience. Users who install a PWA
+have better engagement metrics than typical visitors, including more repeat
+visits, longer time on site and higher conversion rates, often at parity with
+native app users on mobile devices.
+
+If your PWA has use cases where it's helpful for a user to install your app,
+for example if you have users who use your app more than once a week, you
+should be promoting the installation of your PWA within the web UI of your
+app.
+
+{% Aside %}
+See [Enabling installed experiences](/customize-install/) for the code required
+to implement a PWA installation promotion.
+{% endAside %}
+
+## PWA install promotion best practices {: #best-practices }
+
+Some best practices apply no matter what promotional patterns you're using
+on your site.
+
+1. Keep promotions outside of the flow of your user journeys. For example,
+   in a PWA login page, put the call to action below the login form and submit
+   button. Disruptive use of promotional patterns reduce the usability of your
+   PWA and will negatively impact your engagement metrics.
+2. Include the ability to dismiss or decline the promotion. Remember the
+   user's preferences if they do this and only re-prompt if there's a change
+   in the user's relationship with your content such as if they signed in or
+   completed a purchase.
+3. Combine more than one of these techniques in different parts of your PWA,
+   but be careful not to overwhelm or annoy your user with install promotion.
+   **Remember rule 1!**
+4. Only show the promotion when you detect that the `beforeinstallprompt`
+   event has been fired.
+
+## Automatic browser promotion {: #browser-promotion }
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="pwa-omnibox-install.jpg" alt="Install button in the omnibox for Hulu.com">
+  <figcaption class="w-figcaption">
+    Install button in the omnibox provided by the browser.
+  </figcaption>
+</figure>
+
+When [certain criteria](/install-criteria/) are met, the browser will
+automatically indicate to the user that your Progressive Web App is
+installable. For example, desktop Chrome shows an install button in the omnibox.
+
+<div class="w-clearfix">&nbsp;</div>
+
+{% Aside %}
+Chrome for Android will show a mini-infobar to the user, though this can be
+prevented by calling `preventDefault()` on the `beforeinstallprompt` event.
+If you do not call `preventDefault()`, the banner will be shown the first time
+a user visits your site and it meets the installability criteria on Android,
+and then again after approximately 90 days.
```

Comment:
Maybe link to the appropriate heading in #2177?

Reply:
There's nothing great to link to right now, might require a change later.